.travis/apisix_cli_test.sh

```diff
@@ -72,3 +72,47 @@ done
 
 sed -i '/dns_resolver:/,+4s/^#//'  conf/config.yaml
 echo "passed: system nameserver imported"
+
```

Comment:
I think we should add test cases in `t/admin/`.
like https://github.com/apache/incubator-apisix/blob/master/t/admin/stream-routes-disable.t#L37 to change `config.yaml`, and call https request like https://github.com/apache/incubator-apisix/blob/master/t/plugin/redirect.t#L608

Reply:
I think the current implementation is better.

The `conf/nginx.conf` is generated by `conf/config.yaml`, so we use `CLI` to test this feature is better.

The `conf/nginx.conf` in `Test:Nginx` was generated by `t/APISIX.pm`, it is very different for a real user.

Reply:
@moonming wait for your response

Reply:
I think the test coverage of add test case in shell is too low, e.g. I don't know what happens if I open https but send an http request, and it not test that the host is incorrect, it's basically the same as not testing at all.
Maybe we can merge this PR first, but test cases must added later.